cves/CVE-2013-2277.yml

```diff
@@ -127,7 +137,9 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The lifetime of this vulnerability was about 2 years. It went through big
+    changes as there were lines of code that handles the input cases. 
```

Comment:
Could you describe some of the change? e.g. refactoring or code modification.

Reply:
I will describe some of the change by giving some actual examples from the commit which includes out of array access handling, decoding of images when compression changes, buffer overflow, etc.